src/px-script/main/topbar/NavMenu.js

```diff
@@ -56,7 +56,7 @@ export default class NavMenu {
         this.navMenuElement.classList.add("d-block");
         setTimeout(() => {
             this.navMenuElement.classList.add("topbar-nav-open");
-        }, 10);
+        }, 100);
```

Comment:
Why timeout and why have it changed?

Reply:
It gives time for the animation to run, not sure how it works... Maybe @erikhallaraaker can clarify. I increased the timeout because if you clicked the open button fast then the animation would cancel.

I can see that I have mixed two different pull requests, i'll seperate the sheet fix and make a new PR to fix the issue @VikingTristan found.